### DISCUSSION TITLE
Option to view raw as default

### DISCUSSION THREAD

**Discussion by bmocanu** [community/community#20021] [Mobile]
A toggle/dropdown in the Settings - Code view section to select the default code view (raw or processed/rendered) would be great. Right now, Markdown files are displayed as rendered content and there is an option to "View raw". It would be nice to have this new toggle in Settings to display by default all files as raw and have the option in the ellipsis menu for "View rendered" (or some other name).

Thanks a lot!

PS: I think I forgot to label this discussion as for mobile. This is for the mobile app of Github. Sorry!

**Comment by iskae** [comment]
Hi @bmocanu 👋 

Thanks for reaching out and for your first ever feedback on GitHub 🎉 

I think that it would be a nice addition. I have created an internal issue for your request 👍 Please stay tuned for our future releases 🙇 

Cheers!
